models/monitoring.go

```diff
@@ -51,7 +51,7 @@ type StationOverviewData struct {
 	Tags                  []Tag                `json:"tags"`
 	Leader                string               `json:"leader"`
 	Followers             []string             `json:"followers"`
-	SchemaName            string               `json:"schema_name"`
+	Schema                SchemaDetails        `json:"schema"`
```

Comment:
change the type to StationOverviewSchemaDetails which will contain 3 fields name, versionNumber, updatesAvailable